docs/schema.md

```diff
@@ -854,11 +859,51 @@ A, B, C, D, E, F`. This is the desired behaviour in most cases.
 If `limit` is set to `Y`, the list of affected commits will be `X, A, B, C`. This
 is equivalent to `git rev-list X..Y` (but including `X` and excluding `Y`).
 This may be useful if the scope of a vulnerability entry is limited to a small
-set of linear branches. Multiple `"limit"` events may be specified for each
+set of linear branches. Multiple `limit` events may be specified for each
 branch -- each expands the scope of the git commit graph to cover.
 
 Note that we did not specify a `fixed` event here as `limit` makes it redundant.
 
+#### `last_affected` vs `fixed` example
+The difference between `last_affected` and `fixed` is subtle and these examples will 
```

Comment:
Perhaps a better way to open this is:

The difference between `last_affected` and `fixed` has additional implications around false negatives.

Reply:
I took the idea, but rephrased the sentence. Thanks Oliver!